DATA_README.md

```diff
@@ -0,0 +1,21 @@
+# Cumulus - Data model
+
+Cumulus uses DynamoDB for database storage. While DynamoDB is a schemaless database storage system, Cumulus enforces schemas on record creation and update using the [`ajv` package](https://github.com/epoberezkin/ajv).
+
+The data models managed by Cumulus and their schema definitions can be found in the [Cumulus source code](./packages/api/models/schemas.js).
+
+## Managing data schemas
+
+By default, all changes to Cumulus data models should be **backwards-compatible with all previous versions of the data models since the last data migration (if any)**. By "backwards-compatible", we mean:
+
+- Cumulus API endpoint responses have the same structure and properties
+
+Thus, any changes to the underlying data model schemas would require corresponding functions to translate between the updated schemas and all previous versions of the schemas so that data read from the models would conform to the same format.
+
+Backwards compatibility is required because all data that Cumulus manages must be readable and usable in a consistent format. Furthermore, given that Cumulus is actively being used in operational environments which are continuously ingesting petabytes of data, if backwards compatibility were not enforced it would not be realistic to assume that all previously ingested data would be re-ingested to conform to the updated schemas.
```

Comment:
I'm struggling to determine for myself **why** Cumulus data must be output in a consistent format. Why can't our API endpoint response change?

Reply:
I think we could change the API response, but we want to minimize those changes to minimize the impact to those who may have coded around it. If we had an API that was just out there we'd have to be  backwards compatible or go to v2 of the API. But since everyone manages their own deployments we could release changes in a "major" version release, since it would be breaking. Originally (2 years ago) we were talking about API versioning, but I'm not sure that is so necessary.

The concern for this ticket is if we changed something in `api/models/` and workflows were in flight and they were expecting say collection to be in a certain structure and the structure changed and the workflow was using lambdas that had not been updated.